.chloggen/kubelet-client.yaml

```diff
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
```

Comment:
Is it a `breaking` change or rather a `bug_fix`?

Reply:
I would consider the current behavior wrong, which needs to be fixed.

Reply:
Agreed